app/packages/jaeger/src/components/Monitor.test.tsx

```diff
@@ -0,0 +1,245 @@
0Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:26:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:27:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:28:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:29:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:30:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:31:00Z"},{"gaugeValue":{"doubleValue":2333.333333333333},"timestamp":"2023-03-28T06:32:00Z"},{"gaugeValue":{"doubleValue":2000},"timestamp":"2023-03-28T06:33:00Z"},{"gaugeValue":{"doubleValue":2000},"timestamp":"2023-03-28T06:34:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T06:56:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T06:57:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T06:58:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T06:59:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:00:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:01:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:02:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:03:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:04:00Z"},{"gaugeValue":{"doubleValue":30},"timestamp":"2023-03-28T07:05:00Z"}]},{"labels":[{"name":"service_name","value":"csvupload"},{"name":"operation","value":"storage.put "}],"metricPoints":[{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:31:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:32:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:33:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:34:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:35:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:36:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:37:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:38:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:39:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T09:40:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:17:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:18:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:19:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:20:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:21:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:22:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:23:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:24:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:25:00Z"},{"gaugeValue":{"doubleValue":375},"timestamp":"2023-03-27T14:26:00Z"}]}],"name":"service_operation_latencies","type":"GAUGE"}`,
```

Comment:
Can we put this into a json file (`test-data.json`) and import it here?

Reply:
Done